awx/ui_next/src/components/Search/Search.jsx

```diff
@@ -215,8 +231,9 @@ class Search extends React.Component {
           ({ key, name, options, isBoolean, booleanLabels = {} }) => (
             <DataToolbarFilter
               chips={chipsByKey[key] ? chipsByKey[key].chips : []}
-              deleteChip={(unusedKey, val) => {
-                onRemove(chipsByKey[key].key, val);
+              deleteChip={(unusedKey, chip) => {
+                const [columnKey, ...value] = chip.key.split(':');
```

Comment:
does `unusedKey` here equal `chip.key`?

Reply:
no, it is the key under which that chip is grouped:

unusedKey == "Type"
chip == {key: "or__scm_type:", node: {…}}